networkx/algorithms/community/modularity_max.py

```diff
@@ -32,6 +34,9 @@ def greedy_modularity_communities(G, weight=None, resolution=1):
        The name of an edge attribute that holds the numerical value used
        as a weight.  If None, then each edge has weight 1.
        The degree is the sum of the edge weights adjacent to the node.
+    n_communities: int
+       desired number of communities, defaults to 1 and falls back to 1 if
+       the given number is larger than the initial amount of communities
```

Comment:
```suggestion
    n_communities: int
       Desired number of communities: the community merging process is
       terminated once this number of communities is reached. Must be between 1 and the
       total number of nodes in `G`. Default is ``1``, meaning the community
       merging process runs to completion.
```

Just some wording suggestion(s) to try to illustrate exactly what this parameter does. I'm not sure the wording I proposed is great, so feel free to improve --- my goal was to try to really highlight what this parameter does.